eppo/src/main/java/cloud/eppo/android/dto/Allocation.java

```diff
@@ -1,20 +1,25 @@
 package cloud.eppo.android.dto;
 
 import java.util.List;
-import com.google.gson.annotations.SerializedName;
 
 public class Allocation {
-    @SerializedName("percentExposure")
     private float percentExposure;
 
-    @SerializedName("variations")
     private List<Variation> variations;
 
     public float getPercentExposure() {
         return percentExposure;
     }
 
+    public void setPercentExposure(float percentExposure) {
```

Comment:
Now that we are not using automagic, we need explicitly-defined setters for our DTOs